Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.schema.mof

```diff
@@ -1,7 +1,5 @@
-[ClassVersion("1.2.0")]
+[ClassVersion("1.0.0")]
```

Comment:
The usual: pBuild; "make test", but these probably won't reveal any issues because of DSC changes.

Scenarios to test:
1. Machine that previously had nxOMSAutomationWorkerResource 1.2 -> does it get the new 1.0 version after it gets published?
2. Does a newly installed agent get 1.0 instead of 1.2 when the resource is already published in the region the agent is connected to?
3. Any resource-specific scenarios you normally test

I realize that several of the above are only possible after this code gets checked in; I would suggest testing them at that point.